README.rst

```diff
@@ -63,6 +63,19 @@ pages. The simplest way to achieve this is to include this in your
 
     <script src="https://assets.ubuntu.com/v1/842d27d3-lazysizes.min.js" async></script>
 
+NOTE: When JS is disabled Chrome will display a broken image and alt text for the lazysizes version which can cause layout issues.
```

Comment:
I'm confused - aren't we displaying the image with `<noscript>` when JS is off?

Reply:
We are, but noscript doesn't prevent the lazysizes image tag being rendered, resulting in the broken non-noscript image and the correct noscript image both being rendered. In chrome the lazysizes image displays a default broken image before the noscript image.

Reply:
Oh I see. Good point.